.ci/jobs/fpm-mbp.yml

```
@@ -13,7 +13,8 @@
           discover-pr-forks-trust: permission
           discover-pr-origin: merge-current
           discover-tags: false
-          notification-context: 'beats-ci'
+          disable-pr-notifications: false
```

A bit confused, do you mean to change the notification? or disable it? If the latter, then

```suggestion
          disable-pr-notifications: true
```